cli/src/pcluster/constants.py

```diff
@@ -263,6 +266,15 @@ class Feature(Enum):
     Feature.CLUSTER_HEALTH_METRICS: ["us-iso"],
 }
 
+SUPPORTED_OSES_FOR_FEATURE = {
+    Feature.INTEL_PYTHON: list(set(SUPPORTED_OSES) - {"centos7", "alinux2"}),
+    Feature.INTEL_ONE_API_BASE_TOOLKIT: list(set(SUPPORTED_OSES) - {"centos7"}),
+    Feature.INTEL_HPC_SPECIFICATION_2018: ["centos7"],
+}
+
+FEATURE_REQUIRING_ADDITION_SPACE = {Feature.INTEL_ONE_API_BASE_TOOLKIT: 30}
```

Comment:
Let's specify the unit of this space. I assume it is GB, but better to add it as a comment or within the name of the variable.